doc/api.md

```diff
@@ -177,29 +185,34 @@ messages. The listener will receive an object with the following structure:
 changes. The listener will receive an object with the following structure:
 ```javascript
 {
-"jid": jid,                 // the JID of the participant that changed his display name
+"id": id,                 // the id of the participant that changed his display name
 "displayname": displayName  // the new display name
 }
 ```
 
 * **participantJoined** - event notifications about new participants who join the room. The listener will receive an object with the following structure:
 ```javascript
 {
-"jid": jid   // the JID of the participant
+"id": id,   // the id of the participant
+"displayName": displayName, // the display name of the participant
+"avatar": avatar // the avatar URL of the participant
```

Comment:
Could you tell me where to look to find the avatar url being passed into the participantJoined event?

Reply:
In API.js function notifyUserJoined.

Reply:
Okay, but notifyUserJoined gets called in conference but I don't see the avatar url being passed in.

Reply:
OK I see now. Yeah I think it's a bug!

Reply:
Oh! So I remembered now! we are receiving the avatar later in jitsi-meet for the remote participants. I have to remove it from the docs only.